tailwind-workspace-preset.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     extend: {
       spacing: {
         'right-help-sidebar': '22.5rem',
+        'icon-size': '0.875rem',
```

Comment:
Instead to use this, why you didn't use text-sm?

Reply:
You can't use a text-size value with width and height.
You can't do `w-text-sm`